### PR TITLE
Add ability to set job labels to SDK

### PR DESCRIPTION
**Pull request by sammcveety**
@dhalperi , can you take a look?

### CODE REVIEW

**Comment by dhalperi** [comment]
Would you mind rebasing on master? My review of this is being corrupted by the other PR that I just merged (since they share changes). I can't figure out how to get `github` to do the diffbase for me.

**Comment by dhalperi** [comment]
nm, looks pretty clean regardless ;)

File: sdk/src/main/java/com/google/cloud/dataflow/sdk/options/DataflowPipelineOptions.java [hunk 1]
```diff
@@ -114,4 +116,10 @@ public String create(PipelineOptions options) {
       return normalizedAppName + "-" + normalizedUserName + "-" + datePart;
     }
   }
+
+  /**
+   * Labels that will be applied to the billing records for this job.
+   */
+  Map<String, String> getLabels();
```
Comment (dhalperi) [inline sdk/src/main/java/com/google/cloud/dataflow/sdk/options/DataflowPipelineOptions.java]: * `@Description` ?

**Comment by sammcveety** [comment]
Done!

**Pull request merged**